docs/docs-content/clusters/edge/edge-configuration/installer-reference.md

```diff
@@ -99,9 +99,21 @@ stylus:
 
 You can change the port that the Edge management console is exposed on. The default port is 5080.
 
-| Parameter        | Description                                                                               |
-| ---------------- | ----------------------------------------------------------------------------------------- |
-| `emcServer.port` | Specifies the port that the Edge management console is exposed on. Default value is 5080. |
+| Parameter               | Description                                                                                                                                                                       |
+| ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `emcServer.port`        | Specifies the port that Local UI is exposed on. Default value is 5080.                                                                                                            |
```

Comment:
We have added `Default Value` as the last column in several of the tables below that have values with a default vale. We could do the same here.